Alpaca.Markets/RestClient.General.cs

```diff
@@ -116,7 +118,8 @@ public Task<IAccount> GetAccountAsync()
                 TimeInForce = duration,
                 LimitPrice = limitPrice,
                 StopPrice = stopPrice,
-                ClientOrderId = clientOrderId
+                ClientOrderId = clientOrderId,
+                ExtendedHours = extendedHours
```

Comment:
According to API specification this field should be set only for day limit orders. Maybe it would be better to rewrite it in this way:

``` cs
            if (type == OrderType.Limit &&
                duration == TimeInForce.Day &&
                (extendedHours ?? false))
            {
                newOrder.ExtendedHours = extendedHours;
            }

```

But if it just ignored by API in all other cases I'm OK with current approach too.

Reply:
The API does return an error, as it should. If a user is trying to specify extended hours for any of the order types we support other than `day`, it would not make any sense. I think if a user specifies "extended hours" with a "gtc" order, they should get an error so they know to code it right. I don't like the idea of silently discarding errors, and don't really see a need to duplicate the logic in the SDK when the API is already capable of detecting the issue.

Reply:
Totally agree!